traveler_app/src/main/java/com/guestlogix/traveler/activities/SettingsActivity.java

```diff
@@ -0,0 +1,88 @@
+package com.guestlogix.traveler.activities;
+
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.guestlogix.traveler.R;
+import com.guestlogix.travelercorekit.utilities.TravelerPrefs;
+
+import static com.guestlogix.travelercorekit.utilities.UrlHelper.isValid;
+
+public class SettingsActivity extends AppCompatActivity {
+
+    private TravelerPrefs travelerPrefs = TravelerPrefs.getInstance(this);
+
+    private TextView sdkEndpointEditText;
+    private TextView authEndpointEditText;
```

Comment:
Should these two be editTexts instead?
private EditText fooBar;

Reply:
Using Android Preferences now